squadise/squadise-server
# Account and profile related APIs

Konnichi wa! 👀🎃
Change log:
- View/Get profile API
- Test for View/Get profile API
- Verify password API
- Edit/Change password API
- Test for edit/change password
- Separate database related functions
- Add dynamic user account updater
